skfda/exploratory/visualization/_boxplot.py

```diff
@@ -249,8 +242,19 @@ class Boxplot(FDataBoxplot):
 
     """
 
-    def __init__(self, fdatagrid, depth_method=ModifiedBandDepth(), prob=[0.5],
-                 factor=1.5):
+    def __init__(
+        self,
+        fdatagrid: FDataGrid,
+        depth_method = ModifiedBandDepth(),
+        prob: Sequence[float] = [0.5],
```

Comment:
Use a tuple instead.

Reply:
how i will manage it? why the tuple? so instead of sequence a tuple of floats, like (0,5, )?

Reply:
It is because a tuple is immutable and a list isn't (so if it was modified, it would be modified for next function calls). Yo can create that tuple as `(0.5,)` (note the comma before the closing parentheses).